auslib/blobs/superblob.py

```diff
@@ -33,13 +33,22 @@ def getInnerHeaderXML(self, updateQuery, update_type, whitelistedDomains, specia
         """
         :return: Header specific to GMP and systemaddons superblob
         """
-        if not self.get("products"):
+        if self.get("blobs"):
             revision = self['revision']
             # In case of systemaddons superblob
             return '    <addons revision=\"%i\">' % (revision)
-        else:
+        elif self.get("products"):
             # In case of GMP superblob
             return '    <addons>'
+        else:
```

Comment:
Is this case even possible? I'm pretty sure either blobs or products must be defined in a SuperBlob.